pierone/api.py

```diff
@@ -116,6 +116,19 @@ def request(url, path, access_token: str = None, not_found_is_none: bool = False
         return r
 
 
+def request_post(url, path, body, access_token: str = None, not_found_is_none: bool = False) -> requests.Response:
```

Comment:
it would be better to extend the existing `request` function to allow specifying method and data